swo_aws_extension/flows/jobs/Invitations_report_creator.py

```
@@ -0,0 +1,315 @@
+import datetime as dt
+import logging
+from io import BytesIO
+
+from azure.storage.blob import BlobSasPermissions, BlobServiceClient, generate_blob_sas
+from mpt_extension_sdk.mpt_http.base import MPTClient
+from mpt_extension_sdk.mpt_http.mpt import _paginated, get_agreements_by_query  # NoQA: PLC2701
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `PLC2701`)

Remove unused `noqa` directive

(RUF100)